tests-and-examples/certs/client1.crt

```diff
@@ -1,15 +1,26 @@
 -----BEGIN CERTIFICATE-----
-MIICTzCCAbgCFGliNNeTKbKSUv7jbi+4PF0yE9DUMA0GCSqGSIb3DQEBCwUAMF8x
-CzAJBgNVBAYTAkNOMRAwDgYDVQQIDAdCZWlqaW5nMRAwDgYDVQQHDAdCZWlqaW5n
-MRAwDgYDVQQKDAdQaW5nQ0FQMQ0wCwYDVQQLDARUaUtWMQswCQYDVQQDDAJDQTAg
-Fw0yMDAzMDcxMTA1NDRaGA8yMDc0MTIwOTExMDU0NFowbDELMAkGA1UEBhMCQ04x
-ETAPBgNVBAgMCFNoYW5naGFpMREwDwYDVQQHDAhTaGFuZ2hhaTEQMA4GA1UECgwH
-UGluZ0NBUDENMAsGA1UECwwEVGlLVjEWMBQGA1UEAwwNZ3JwYy1jbGllbnQtMTCB
-nzANBgkqhkiG9w0BAQEFAAOBjQAwgYkCgYEA1oCOEdTujz8Szi/dNAuRnFBuW2zO
-26/uOfv4EJocVIQ2CSuD1bgGIgUeiURPmEcaawy6jKlP4Rk2Uon1zZ+DquOHx2vX
-Xq/TEX6LTCZDdLLDZ0M0gshq0/0MVvSQa1uNLG+yOdiotCo44aWwVqOkzmRtnuFt
-OZ0+oKnYQojIG1kCAwEAATANBgkqhkiG9w0BAQsFAAOBgQCg2DOBoq6Ty8RL08KG
-aBQLz+jWJTdD8eZOfJvffa1j5NntPHWa1Ei4giQtamChoCJ5EAAp7Qx2JLE8OmQD
-gFlHzsMIgpaCpLwYCYFI5IDm/7r2XNpYTGAtKmsRO1ahNzoBgt4WoejRZUy9BNml
-f+SNKDL/xvIX6V5ZSka1bOrT1w==
+MIIEezCCAmOgAwIBAgIUZaC7vTgYBFXsTUWwCiXR8IMANNMwDQYJKoZIhvcNAQEL
```

Comment:
Why changing the certificate?

Reply:
I have tested using cert with just CN, it's OK

Reply:
So the changes are not necessary then?